helm-chart/renku/values.yaml

```diff
@@ -230,7 +230,7 @@ postgresql:
 
   image:
     repository: bitnami/postgresql
-    tag: 9.6
+    tag: 11
```

Comment:
should we go for `11.11.0` (latest)?
Otherwise I'd pin this in all of our values files...